### PR TITLE
Added redis cluster config params

### PR DESCRIPTION
**Pull request by chillaq**
# .NET SDK

## Extra Notes

### CODE REVIEW

File: src/Splitio/Domain/AdapterType.cs [hunk 1]
```diff
@@ -2,6 +2,7 @@
 {
     public enum AdapterType
     {
-        Redis //Redis by default
+        Redis, //Redis by default
+        RedisCluster
```
Comment (sanzmauro) [inline src/Splitio/Domain/AdapterType.cs:6]: this is not necessary. please remove it.

File: src/Splitio/Domain/CacheAdapterConfigurationOptions.cs [hunk 2]
```diff
@@ -15,6 +16,8 @@ public class CacheAdapterConfigurationOptions
         public string UserPrefix { get; set; }
         public TlsConfig TlsConfig { get; set; }
         public int? PoolSize { get; set; }
+        public List<string> ClusterNodes { get; set; }
```
Comment (sanzmauro) [inline src/Splitio/Domain/CacheAdapterConfigurationOptions.cs:19]: I suggest to create `ClusterNodes` class with `endpoints` and `keyHastTag` properties.

**Review by sanzmauro — CHANGES_REQUESTED** [review]

File: src/Splitio/Domain/AdapterType.cs (hunk 1 above)
Comment (chillaq) [reply src/Splitio/Domain/AdapterType.cs:6]: will do

File: src/Splitio/Domain/CacheAdapterConfigurationOptions.cs (hunk 2 above)
Comment (chillaq) [reply src/Splitio/Domain/CacheAdapterConfigurationOptions.cs:19]: will do

File: src/Splitio/Domain/ClusterNodes.cs [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+    {
+        public string KeyHashTag {  get; set; }
+        public List<string> EndPoints { get; set; }
+
+        public ClusterNodes(List<string> endPoints, string keyHashTag) 
+        {
+            EndPoints = endPoints;
+            KeyHashTag = keyHashTag;
+        }
+
```
Comment (sanzmauro) [inline src/Splitio/Domain/ClusterNodes.cs:19]: please remove empty line

**Review by sanzmauro — APPROVED** [review]
Remove the empty line before merging please 🙏

**Pull request merged**